src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -40,23 +45,39 @@ class EventOrGroupHeader extends React.Component {
     }
   }
 
-  render() {
-    let {className, hideLevel, hideIcons, orgId, projectId, data} = this.props;
-    let cx = classNames('event-issue-header', className);
+  getTitle() {
+    let {hideLevel, hideIcons, includeLink, orgId, projectId, data} = this.props;
     let {id, level, groupID} = data || {};
     let isEvent = !!data.eventID;
-    let url = `/${orgId}/${projectId}/issues/${isEvent ? groupID : id}/${isEvent ? `events/${data.id}/` : ''}`;
+
+    let props = {};
+    let Wrapper;
+    if (includeLink) {
+      props.to = `/${orgId}/${projectId}/issues/${isEvent ? groupID : id}/${isEvent ? `events/${data.id}/` : ''}`;
+      Wrapper = Link;
+    } else {
+      Wrapper = 'span';
```

Comment:
is there another way to do this? looks kind of weird/wrong, but it's the only thing i could come up with

Reply:
I think that's good.

If it looks weird, you could use a ternary and hope we don't need more conditions in the future